### DISCUSSION TITLE
Scrolling down a lot causes issues

### DISCUSSION THREAD

**Discussion by Moosems** [community/community#56429] [Mobile] [answered]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.105.0 (94356885)
Device: iPhone14,7 (iOS 16.2)
</details>

https://github.com/community/community/assets/95927277/e07e2223-5404-449b-81fb-83a0fef284a6

**Comment by eliperkins** [comment, marked as answer]
Hey @Moosems, thanks for reporting this bug! It definitely looks like behavior that shouldn't be happening, so I've filed an issue on our internal repo with steps to reproduce this bug. I'll let you know when the bug has been fixed. 🙌

**Reply by Moosems** [reply]
Thanks! Love the hard work your team is doing :).
